docs/source/user/fast.farm/InputFiles.rst

```diff
@@ -624,11 +625,58 @@ If DEFAULT is used, then **Mod_Projection=2** when **Mod_Wake=2**,
 and **Mod_Projection=1** otherwise.
 
 
-
-**OutAllPlanes** [-] Output all wake planes in VTK at all time steps. 
-Note: this option requires intensive writing to disk and will drastically slow down the simulation.
+Wake-Added Turbulence (WAT)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The wake-added turbulence model is described in :numref:`FF:WAT`.
+
+**WAT** [switch] Select whether the wake-added turbulence is included.
+There are three options:
+0) no wake-added turbulence,
+1) use predefined turbulence box for the background turbulence,
+2) use a user defined turbulence box.
+When **WAT=1**, the number of points of the box are inferred from the filename, 
+and the dimensions in each directions are taken as :math:`dx=dy=dz=0.03*\text{RotorDiamRef}`.
+
+**WAT_BoxFile**: [quoted string]
+Filepath to the file containing the u-component of the turbulence box 
+(either predefined or user-defined). This turbulence box is expected to be 
+in the Mann box format. 
+The filename should be of the form "Label_1024x32x16.u" where the extension ".u" 
+is required. FAST.Farm will assume that files with extensions ".v" and ".w" are present,
+and these files will also be read. 
+When **WAT=1**, the three digits separated by "x" in the filename are assumed to be 
+the number of points in the x y and z dimension of the box.
+
+**WAT_NxNyNz**: [three integers, comma separated] 
+Number of points in the x, y, and z directions of the WAT_BoxFile.
+These are used only if WAT=2. 
+
+**WAT_DxDyDz**: [three floats, comma separated] 
+Distances (in meters) between points in the x, y, and z directions of the WAT_BoxFile 
+These are used only if WAT=2.
+When **WAT=1** the dimensions in each directions are taken as :math:`dx=dy=dz=0.03*\text{RotorDiamRef}`.
+
+**WAT_ScaleBox**: [flag]  
+When set to True, the input turbulence box is scaled so that it has zero mean and unit standard deviation at every node.
 DEFAULT is False.
 
+**WAT_k_Def** [float]
+Parameter for the influence of the wake deficit in the wake-added turbulence 
+[>=0.0, no upper bound]
+Currently uncalibrated, no default value. Anticipated value is expected to be around 0.5.
+
+**WAT_k_Grad** [float]
+Parameter for the influence of the radial velocity gradient of the wake deficit in the wake-added turbulence 
+[>=0.0, no upper bound].
+Currently uncalibrated, no default value. Anticipated value is expected to be around 0.5.
+
+**WAT_D_BrkDwn** [float, dimensionless]
+Downstream distance in rotor diameter after which WAT scaling has reached 100% capacity.
+DEFAULT is 1 (i.e. 1D)
```

Comment:
We've discussed potentially replacing input `WAT_D_BrkDwn` with one of the filter functions involved in the eddy viscosity formulation.  I would guess F_vShr(x) would make sense because it represents the development of turbulent stresses generated by the wake shear layer, which seems to be what `WAT_D_BrkDwn` is trying to achieve.  The only issue I see is the DEFAULT values of `C_vShr_DMin`, `C_vShr_DMax`, `C_vShr_FMin`, and `C_vShr_Exp` used to define F_vShr(x) result in a filter function that does not approach unity until 25D.  This would impact the calibration of `WAT_k_Def` and `WAT_k_Grad`.

Reply:
I think it might be risky to rely on the same inputs indeed. 

But it's true that we could use the function `EddyFilter`, and call it with `C_Dmin=0` and `C_Dmax = WAT_D_BrkDwn`
As for the `FMin` and the exponent, we could use the values from the input file, but they might represent something a bit different, maybe it's best to hardcode them to say `FMin=0` and `Exp=1`.

Reply:
I'm wondering if we should hold off on deciding this until the calibration is further along.  I recall thinking before that the calibration may require different values at different downstream distances, which may work well if we have a filter function that is downstream-distance-dependent.